src/utils/discord.ts

```diff
@@ -0,0 +1,29 @@
+import { DiscordAPIError, Message, MessageCreateOptions } from 'discord.js'
+
+/**
+ * Sends a direct message (DM) to the author of a message if they have enabled DMs.
+ *
+ * @param {Message} message - The original message object.
+ * @param {MessageCreateOptions} payload - The payload of the message to be sent.
+ */
+export async function sendDm(
```

Comment:
i don’t recommend naming it `discord.ts` because

1. all everything related to discord will end up here, making this file bloat
2. there is already `discord.js`

recommend just calling it `sendDm.ts`

> Aside: Make a util directory and keep different utilities in different files. A single util file will always grow until it is too big and yet too hard to split apart. Using a single util file is unhygienic.
>
> — https://programmingisterrible.com/post/139222674273/write-code-that-is-easy-to-delete-not-easy-to